integration/airflow/openlineage/airflow/facets.py

```diff
@@ -27,3 +28,21 @@ def from_task(cls, task):
 @attr.s
 class AirflowRunArgsRunFacet(BaseFacet):
     externalTrigger: bool = attr.ib(default=False)
+
+
+@attr.s
+class UnknownOperatorInstance:
```

Comment:
I would use `UnknownOperatorClass` for the facet, as the `name` is set to the name of the operator class not currently supported.

Reply:
It does use the class name as the name f the item, but it's recording the fields of the instance, not just generic class properties, so... 🤷🏽‍♂️

Reply:
Ahh, makes sense 👍